app/models/concerns/foreman_bootdisk/compute_resources/proxmox.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+module ForemanBootdisk
+  module ComputeResources
+    module Proxmox
+      def capabilities
+        super + [:bootdisk]
+      end
+
+      def iso_upload(iso, vm_uuid)
+        server = find_vm_by_uuid(vm_uuid)
+        server.ssh_options = { password: fog_credentials[:proxmox_password] }
+        server.ssh_ip_address = server.node_id
+        server.username = client.credentials[:user].split('@').first
+        server.scp_upload(iso, '/var/lib/vz/template/iso/')
+        server.reload
+        storage = storages(server.node_id, 'iso')[0]
+        storage.volumes.any? { |v| v.volid.include? File.basename(iso) }
+      end
+
+      def iso_attach(iso, vm_uuid)
+        server = find_vm_by_uuid(vm_uuid)
+        storage = storages(server.node_id, 'iso')[0]
+        volume = storage.volumes.detect { |v| v.volid.include? File.basename(iso) }
+        disks = server.disks.map {|disk| disk.split(":")[0] }.join(";")
```

Comment:
```suggestion
        disks = server.disks.map { |disk| disk.split(":")[0] }.join(";")
```
Codestyle nitpick :grin: I would expect rubocop to complain about this, but it is configured here to basically check nothing :monocle_face: 
Regardless, I guess it makes sense to have consistent formatting :wink: